Opentherm gw improvements

Martidjen opened the pull request:
## Proposed change
<!-- 
    Describe the big picture of your changes here to communicate to the
    maintainers why we should accept this pull request. If it fixes a bug
    or resolves a feature request, be sure to link to that issue in the 
    additional information section.
-->
Making some improvements to the Opentherm Gateway integration:
- Dividing temperature precision into precision for current temperature and precision for setpoint temperature
- Added option for setpoint override mode, can be set to temporary or constant (some thermostats work better with constant mode)


## Type of change
<!--
    What types of changes does your PR introduce to our documentation/website?
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR.
-->

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/47484
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist
<!--
    Put an `x` in the boxes that apply. You can also fill these out after
    creating the PR. If you're unsure about any of them, don't hesitate to ask.
    We're here to help! This is simply a reminder of what we are going to look
    for before merging your code.
-->

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

dshokouhi committed:
Blog post for the android releases in 2021 Q1 (#16837)

balloob committed:
Release 2021.3.1

dependabot[bot] committed:
Bump ruby/setup-ruby from v1.65.0 to v1.66.0 (#16854)

adrum committed:
2021-3 Z-Wave JS version fixed (#16866)

I noticed the Z-WaveJS version was missing the patch version.

klaasnicolaas committed:
✨ The 11th community highlight! (#16859)

ludeeus committed:
Add installation type in the title of common tasks (#16872)

nicolalandro committed:
Update mycroft.markdown (#16873)

example message

frenck committed:
Tweaks to HomeKit documentation (#16875)

Martidjen committed:
Updating opentherm_gw.markdown

Updating opentherm_gw.markdown before starting PR for some improvements on the opentherm_gw integration

homeassistant commented:
Hi @Martidjen, @nicolalandro, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16878).

Once you do that we will be able to review and accept this pull request.

Thanks!

Martidjen committed:
Splitting the parent PR

Splitting the parent PR into multiple ones, one per new feature. Splitting the documentation PR as well

homeassistant commented:
Hi @nicolalandro, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#16878).

Once you do that we will be able to review and accept this pull request.

Thanks!

Martidjen commented:
Looks like i've been branching from the current branch while trying to merge to next.. Let me start over
